Automattic/wp-calypso
# Pricing page: update bundle product images to match new product icons.

#### Proposed Changes
This PR updates the current bundle hero images to match new product icons. Additionally, this PR fixes a minor issue w/ breakpoint inconsistency.

**old image**
<img width="1208" alt="Screen Shot 2023-01-23 at 5 29 25 PM" src="https://user-images.githubusercontent.com/56598660/214006384-41f075c8-f19c-486c-99e0-b24be5f646d1.png">

**new image**
<img width="1206" alt="Screen Shot 2023-01-23 at 5 29 12 PM" src="https://user-images.githubusercontent.com/56598660/214006110-cd2d3e08-c164-4bcc-8908-5cf1e5e57a0b.png">


#### Testing Instructions

Run this branch by following the steps below (or you can use the Jetpack Cloud live link commented on this PR)
    * Run `git fetch && git checkout update/pricing-page-bundle-icons`
    * Run `yarn start`

1. Goto pricing page http://jetpack.cloud.localhost:3000/pricing or use the Jetpack cloud live link and append `/pricing`
2. Confirm that the bundle hero images are updated w/ the new icons.
3. Try to resize your browser and confirm the variant images (sm, md, lg) are displayed properly according to screen size.
<img width="417" alt="Screen Shot 2023-01-23 at 5 02 01 PM" src="https://user-images.githubusercontent.com/56598660/214006917-6f263a9a-9a2c-42e7-8ba3-cfe15787f90d.png"> <img width="483" alt="Screen Shot 2023-01-23 at 5 01 50 PM" src="https://user-images.githubusercontent.com/56598660/214007080-89f7ca90-48a7-4873-9559-2599df59163d.png"> <img width="596" alt="Screen Shot 2023-01-23 at 5 01 43 PM" src="https://user-images.githubusercontent.com/56598660/214007101-187a75e1-297e-4c77-bc5f-f2da4e0b64f2.png">

#### Pre-merge Checklist





Related to  1202858161751496-as-1203766452274876

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203766452274876
